recipes/image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "image-meta-tag" %}
+{% set version = "0.6.3" %}
+{% set giturl = "https://github.com/SciTools-incubator/image-meta-tag" %}
+{% set sha256 = "8670b421aa55102b4e7de6b845bc655f9d35e8659c7fd2a4f326fa4347b21f99" %}
+
+package:
+  name: image-meta-tag
+  version:  {{ version }}
+
+source:
+  git_url: {{ giturl }}
+  git_tag: {{ version }}
+  fn: {{ version }}.tar.gz
+  url: {{ giturl }}/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
I think `{{name}}` should be here. The link I'm seeing is:

https://github.com/SciTools-incubator/image-meta-tag/archive/0.6.3.tar.gz

Reply:
Thanks, you're right, the {{ name }} is superfluous as it's in the {{ giturl }}.